src/api/apps/articles/model/index.js

```diff
@@ -7,7 +7,7 @@ import async from "async"
 import { cloneDeep } from "lodash"
 import schema from "./schema.coffee"
 import Joi from "../../../lib/joi.coffee"
-import retrieve from "./retrieve.coffee"
+import { toQuery } from "./retrieve.js"
```

Comment:
Do we need an extension here? Also I'd assume that for coffee files we need to use a const like on line 13

Reply:
Yeah we could probably get rid of the `.js` extension. 

When you are referring to the const are you meaning the import on line 9 needs to be change because it's a coffeescript file?

Reply:
Yeah, we need to use `require` on all `.coffee` files, `import` works only with js and ts